docs/mkdocs/docs/api/macros/nlohmann_define_type_with_names.md

```diff
@@ -33,14 +33,14 @@ For further information please refer to the corresponding macros without `WITH_N
 :   name of the base type (class, struct) `type` is derived from (used only in `DEFINE_DERIVED_TYPE` macros)
 
 `json_member_name` (in)
-:   used in named conversion macros, must be provided for each member variable and will be used as a member variable name in the resulting json
+:   json name that will be used for the next member variable
```

Comment:
"json" -> "JSON"?

What is a "json name"?

Reply:
How else would you call "a name which will be used for the corresponding value inside the JSON"?

Reply:
> How else would you call "a name which will be used for the corresponding value inside the JSON"?

The string that will be used as the name of the object for the next value.